### PR TITLE
Djsalman feature dynamodb to sns

### PR DESCRIPTION
**Pull request by salmandjing**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cfn-dynamodb-sns/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+   ```
+   git clone https://github.com/aws-samples/serverless-patterns
+   ```
+1. Change directory to the pattern directory:
+   ```
+   cd _patterns-model
```
Comment (mavi888) [inline cfn-dynamodb-sns/README.md:24]: update with the right directory

File: cfn-dynamodb-sns/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   - Enter the desired AWS Region
+   - Allow SAM CLI to create IAM roles with the required permissions.
+
+   Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+1. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
+
+## How it works
+
+Explain how the service interaction works.
```
Comment (mavi888) [inline cfn-dynamodb-sns/README.md:42]: please elaborate

File: cfn-dynamodb-sns/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Testing
+
+Provide steps to trigger the integration and show what should be observed if successful.
```
Comment (mavi888) [inline cfn-dynamodb-sns/README.md:46]: please elaborate

File: cfn-dynamodb-sns/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Cleanup
+
+1. Delete the stack
+   ```bash
+   aws cloudformation delete-stack --stack-name STACK_NAME
```
Comment (mavi888) [inline cfn-dynamodb-sns/README.md:52]: fix to sam delete

File: cfn-dynamodb-sns/example-pattern.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
+{
+  "title": "DynamoDB Streams to trigger SNS messaging",
+  "description": "Create an asynchronous DynamoDB Streams trigger to SNS.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "**NOTE** - Edit the CloudFormation template in a Code Editor like Visual Studio Code and replace “Endpoint: abcd@gmail.com”, with your own personal email.",
```
Comment (mavi888) [inline cfn-dynamodb-sns/example-pattern.json:10]: please use sam parameters for this

File: terraform-amplify/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (mavi888) [inline terraform-amplify/.gitignore:1]: remove this whole folder from the PR

**Comment by mavi888** [comment]
This PR has a lot of weird folders - please clean it up.

Also this project is not using AWS SAM, just raw CF, please use SAM, CDK or other supported frameworks for serverless Land.

Please complete the README File and improve the example-template.

**Comment by mavi888** [comment]
@salmandjing This pattern will be closed next week if there are no updates in this PR. 
If closed, reopen it again later on with the fixes applied. 

Thanks,
Marcia

**Comment by salmandjing** [comment]
@mavi888 I'm learning how to template this as a CDK. Thanks

**Comment by mavi888** [comment]
@salmandjing i will close this PR as is full of weird commits and files that dont make sense. Also the README File is not yet updated. Please submit this again following the instructions and with a clenear PR so it can get approved.

**Pull request closed without merging**